presto-docs/src/main/sphinx/release/release-342.md

```diff
@@ -0,0 +1,49 @@
+# Release 342 (XX Sep 2020)
+
+## General Changes
+
+* Improve performance of queries that use decimal type. ({issue}`4886`)
+* Improve query performance by adding support for dynamic filtering and dynamic
+  partition pruning to semi-join relational operator. ({issue}`5017`)
+* Improve dynamic partition pruning and query performance by reducing latency of dynamic filters collection. ({issue}`4988`)
+* Improve join performance when cost-based optimizer has missing or inaccurate stats. ({issue}`5141`)
+* Enable connectors to wait for dynamic filters derived from replicated join before generating splits. ({issue}`4685`)
```

Comment:
I think this one requires explanation what it actually means for the user.

Reply:
is this a SPI change? @sopel39

Reply:
this should be in SPI notes